README.rst

```diff
@@ -20,15 +20,35 @@ Super CSV is a stand alone library that can be used for CSV management, both syn
 
 Testing
 -------
-
 ::
+
     $make requirements
-Will install all pre-requisuites.
 
-::
+Will install all pre-requisuites. ::
+
     $make test
+
 Runs the unit tests in local environment.
 
+Using with Docker Devstack
+--------------------------
+Prerequisite: Have your Open edX https://github.com/edx/devstack properly installed.
+Note: When you see "from inside the lms" below, it means that you've run ``make lms-shell`` from your devstack directory and are on a command prompt inside the LMS container.
+
+#. Clone this repo into ../src/ directory (relative to your "devstack" repo location). This will mount the directory in a way that is accessible to the lms container. Since this is a standalone library, there should be no need to manually perform a pip-install step in the lms-shell.
+
+#. Setup dev environment::
+
+    cd /edx/src/super-csv
+    virualtenv super-csv-env
+    source super-csv-env/activate
```

Comment:
There are other tools for setting up/managing python virtual environments, so you may want to add a little more natural language text explaining what you're doing here & why, so that folks using their tools can get a sense of how to adapt this guidance/whether they need to.

Reply:
While that's true, our docker images come with `virtualenv` installed on them, so it's likely the only tool to be used for this purpose within the realm of devstack.  But I agree that having a short preamble along the lines of "Create a python virtual environment to install the dependencies that are strictly for `super-csv`..." would be helpful.